libkbfs/block_retrieval_queue.go

```diff
@@ -213,6 +215,8 @@ func (brq *blockRetrievalQueue) notifyWorker(priority int) {
 func (brq *blockRetrievalQueue) CacheAndPrefetch(ctx context.Context,
 	ptr BlockPointer, block Block, kmd KeyMetadata, priority int,
 	lifetime BlockCacheLifetime, hasPrefetched bool) (err error) {
+	didUpdateCh := make(chan struct{})
+	donePrefetch := hasPrefetched
```

Comment:
Yep, figured, sounds good.

I guess technically you could use the encoding options to specify that the encoded name is `HasPrefetched`, while the struct name could be `TriggeredPrefetch`.  But that's probably overkill.  Up to you.